### PR TITLE
Release notes for Bareos 19.2.9

### PR DESCRIPTION
**Pull request by franku**

### CODE REVIEW

File: docs/manuals/source/Appendix/ReleaseNotes.rst [hunk 1]
```diff
@@ -21,6 +21,47 @@ The feature overview for a release are shown at the :ref:`genindex` of this docu
...
+   * - **Database Version**
+     -  2192
+   * - **URL**
+     - https://download.bareos.com/bareos/release/19.2/
+   * - **Release Ticket**
+     - :mantis:`????`
+
+Bugs Fixed
+^^^^^^^^^^
+* stored: fix memory corruption in label.cc
```
Comment (franku) [inline docs/manuals/source/Appendix/ReleaseNotes.rst:48]:
```suggestion
* fix a memory corruption when autolabeling with increased maxiumum block size
* stored: fix memory corruption in label.cc
```

File: docs/manuals/source/Appendix/ReleaseNotes.rst [hunk 2]
```diff
@@ -21,6 +21,47 @@ The feature overview for a release are shown at the :ref:`genindex` of this docu
...
+   * - **URL**
+     - https://download.bareos.com/bareos/release/19.2/
+   * - **Release Ticket**
+     - :mantis:`1260`
+
+Bugs Fixed
+^^^^^^^^^^
+* :mantis:`1281`: Director crashes on memory access violation in CloseMemoryPool() and is_regex()
+* fix a memory corruption when autolabeling with increased maxiumum block size
+* stored: fix memory corruption in label.cc
```
Comment (franku) [inline docs/manuals/source/Appendix/ReleaseNotes.rst:50]:
```suggestion
```

File: docs/manuals/source/Appendix/ReleaseNotes.rst [hunk 3]
```diff
@@ -21,6 +21,56 @@ The feature overview for a release are shown at the :ref:`genindex` of this docu
...
+   * - **URL**
+     - https://download.bareos.com/bareos/release/19.2/
+   * - **Release Ticket**
+     - :mantis:`1260`
+
+VMware Plugin
+^^^^^^^^^^^^^
+Starting with this release the VMware Plugin will be built with VDDK 7.0.1.
+Due to a bug with that version, that may crash bareos_vadp_dumper, the wrapper script will check some prerequisites to make sure the bug will not occur.
+The wrapper script will exit with an error describing the problem if the prerequisites are not met and your job will fail.
```
Comment (pstorz) [inline docs/manuals/source/Appendix/ReleaseNotes.rst:50]: .
```suggestion
If the prerequisites are not met, the wrapper script will exit with an error describing the problem and make the job fail.
```

File: docs/manuals/source/Appendix/ReleaseNotes.rst [hunk 4]
```diff
@@ -21,6 +21,56 @@ The feature overview for a release are shown at the :ref:`genindex` of this docu
...
+The prerequisites are the existence of ``/sys/class/scsi_disk/`` and ``/etc/mtab``.
+This can be achieved easily by loading the sd_mod kernel module and creating ``/etc/mtab`` as a symlink to ``/proc/self/mounts``.
+
+Due to problems with the availability of packages for pyvmomi, the depencency has been removed from the RPM packages.
```
Comment (pstorz) [inline docs/manuals/source/Appendix/ReleaseNotes.rst:54]:
```suggestion
Due to problems with the availability of packages for ``pyvmomi``, the depencency has been removed from the RPM and DEB packages.
```

File: docs/manuals/source/Appendix/ReleaseNotes.rst [hunk 5]
```diff
@@ -21,6 +21,56 @@ The feature overview for a release are shown at the :ref:`genindex` of this docu
...
+While this has to be installed manually now, you don't rely on a RPM packaged version and can now use ``pip`` to install it.
```
Comment (pstorz) [inline docs/manuals/source/Appendix/ReleaseNotes.rst:55]:
```suggestion
The ``pyvmomi`` python dependency now needs to be installed using the ``pip`` command.
```

**Review by pstorz — COMMENTED** [review]
Just some suggestions about wording. Other than that very good.

**Pull request merged**